src/App.tsx

```diff
@@ -25,33 +25,31 @@ function App() {
 	const [encounterInProgress, setEncounterInProgress] =
 		useState<boolean>(false);
 	return (
-
 		<CommandHistoryProvider>
-      <Grommet theme={theme} full>
-        <Page fill="vertical" style={{ height: '100dvh' }}>
-          <AppBar>
-            <Button>
-              <Menu onClick={() => setShow(true)} />
-            </Button>
-            {!encounterInProgress && (
-              <Button
-                primary
-                label="Start Encounter"
-                onClick={() => setEncounterInProgress(true)}
-              />
-            )}
-            <Text size="large">My App</Text>
-          </AppBar>
-          <PageContent style={{ overflowY: 'auto' }}>
-            <PageHeader title="Ta lista potrzebuje integrated wykrywanie gestów swipe & tap" />
-            <InitiativeList />
-          </PageContent>
-          {encounterInProgress && (
-            <EncounterBar endEncounter={() => setEncounterInProgress(false)} />
-          )}
```

Comment:
Why did conditional rendering disappear?

Reply:
To show undo and redo keys always. We should hammer out UI/UX at the end of the MVP phase.